### PR TITLE
Decompile menu_helpers

### PR DESCRIPTION
**Pull request by drifloony**

### CODE REVIEW

File: src/menu_helpers.c [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+    else
+    {
+        MenuPrint(gStringVar1, left, top);
+    }
+}
+
+// unused
+void PrintTriangleCursorWithPalette(u8 left, u8 top, u8 paletteNum)
+{
+    u8 buffer[2];
```
Comment (marijnvdwerf) [inline src/menu_helpers.c]: These `memcpy`s should be fairly trivial to replace with `u8 buffer[] = _("▶$")`; That should place the string in the `.rodata` section of the object file.

Comment (yenatch) [inline src/menu_helpers.c]: What about `PrintStringWithPalette(_("▶"), paletteNum, left, top);`?

Comment (marijnvdwerf) [inline src/menu_helpers.c]: It might, but it probably won't work in `PrintStringWithPalette`

Comment (camthesaxman) [inline src/menu_helpers.c]: You could do `PrintStringWithPalette(&(u8[])_("▶"));`, but that's a bit ugly, and I think the way marijnvdwerf suggested is fine. Just rename `buffer` to something useful like `rightArrow`.

Comment (yenatch) [inline src/menu_helpers.c]: should `preproc` be doing this cast instead?

Comment (marijnvdwerf) [inline src/menu_helpers.c]: (Check the actual function `PrintStringWithPalette` in this PR to see what I meant)

**Pull request merged**